src/main/java/com/user_service/DTOs/SignInResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com.user_service.DTOs;
```

Comment:
Minor: package name in Java would look better as plainly `dtos` or just `dto`. You may also think about separating `request` DTOs from `response` DTOs.